auth/api/logging.go

```diff
@@ -349,6 +349,25 @@ func (lm *loggingMiddleware) CreateDomain(ctx context.Context, token string, d a
 	return lm.svc.CreateDomain(ctx, token, d)
 }
 
+func (lm *loggingMiddleware) DeleteDomain(ctx context.Context, token string, d auth.Domain) (err error) {
```

Comment:
Probably deletion was avoided for now because of cascade consequences - @dborovcanin must confirm. But probably you'll need to remove all resources related to this domain (things, channels, ...) once you decide to remove a specific domain.

Reply:
@drasko This is correct, it requires a careful analysis due to the cascade nature of removal.